kuberos.go

```diff
@@ -300,6 +301,15 @@ func redirectURL(r *http.Request, endpoint *url.URL) string {
 			}
 		}
 	}
+
+	// Redirect includes X-Forwarded-Prefix if exists
+	for h, v := range r.Header {
+		if h == headerForwardedPrefix {
+			for _, prefix := range v {
+				u.Path = prefix
+			}
+		}
+	}
```

Comment:
I'd prefer we only ranged over `r.Header` once. This could be a switch statement:

```go
for h, v := range r.Header {
  switch h {
    case elbHeaderForwardedProto:
      // Existing check for HTTPS protocol
    case headerForwardedPrefix:
      // Set the URL path
  }
}
```

Reply:
OK, makes sense. I change that to a switch